source/back.tex

```diff
@@ -23,6 +23,12 @@ \chapter{Bibliography}
   \doccite{Unicode Text Segmentation} [online].
   Edited by Mark Davis. Revision 35; issued for Unicode 12.0.0. 2019-02-15 [viewed 2020-02-23].
   Available from: \url{http://www.unicode.org/reports/tr29/tr29-35.html}
+\item
+  The Unicode Consortium. Unicode Standard Annex, UAX \#31,
+  \doccite{Unicode Identifier and Pattern Syntax} [online].
+  Edited by Mark Davis. Revision 33; issued for Unicode 13.0.0.
+  2020-02-13 [viewed 2021-06-08].
+  Available from: \url{https://www.unicode.org/reports/tr31/tr31-33.html}
```

Comment:
This whole list item is different that what's at http://wg21.link/P1949R7#wording-for-uax-31-identifiers. It mentions UAX 44 (as is done below), but the wording above mentions UAX 31.